requirements/local.txt

```diff
@@ -13,10 +13,11 @@ django-extensions==1.5.2
 
 # For building documentation
 # http://sphinx-doc.org/
+sphinx_rtd_theme==0.1.7
```

Comment:
Please move outside of Sphinx section and add proper comment with the reasons why this is specified in the first place.